evalml/tests/pipeline_tests/test_pipelines.py

```diff
@@ -384,19 +383,9 @@ def test_describe_pipeline(
         assert component.name in out
 
 
-def test_nonlinear_model_family():
+def test_nonlinear_model_family(example_graph):
```

Comment:
Updated this test to just use our example graph, rather than defining a similar component graph below. As such, new expected model family should be linear, not RF as before :)